assets/js/components/help/HelpMenu.js

```diff
@@ -0,0 +1,104 @@
+/**
+ * HelpMenu component.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { Fragment, useState, useRef, useEffect, useCallback } from '@wordpress/element';
+import { ESCAPE } from '@wordpress/keycodes';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import Button from '../Button';
+import HelpIcon from '../../../svg/help.svg';
+import VisuallyHidden from '../VisuallyHidden';
+import HelpMenuLink from './HelpMenuLink';
+import Menu from '../Menu';
+
+function HelpMenu( { children } ) {
+	const [ menuOpen, toggleMenu ] = useState( false );
+	const menuButtonRef = useRef();
+	const menuRef = useRef();
+
+	useEffect( () => {
+		const handleMenuClose = ( event ) => {
+			if ( menuButtonRef?.current && menuRef?.current ) {
+				// Close the menu if the user presses the Escape key
+				// or if they click outside of the menu.
+				if (
+					( ( 'keyup' === event.type && ESCAPE === event.keyCode ) || 'mouseup' === event.type ) &&
+					! menuButtonRef.current.contains( event.target ) &&
+					! menuRef.current.contains( event.target )
+				) {
+					toggleMenu( false );
+				}
+			}
+		};
+
+		global.addEventListener( 'mouseup', handleMenuClose );
+		global.addEventListener( 'keyup', handleMenuClose );
+
+		return () => {
+			global.removeEventListener( 'mouseup', handleMenuClose );
+			global.removeEventListener( 'keyup', handleMenuClose );
+		};
+	}, [] );
+
+	const handleMenu = useCallback( () => {
+		toggleMenu( ! menuOpen );
+	}, [ menuOpen ] );
+
+	return (
+		<Fragment>
+			<div className="googlesitekit-user-selector googlesitekit-dropdown-menu googlesitekit-help-menu mdc-menu-surface--anchor">
```

Comment:
I noticed the `googlesitekit-user-selector` class here which doesn't really belong except that it removes the dropdown arrow. Let's extract this to a different class to not mix user-selector styles with this element.

Reply:
Sure thing—I think it's like this in another menu this was based off, but good point 👍🏻